irs_lqr/irs_lqr.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
+import time
+
+from irs_lqr.tv_lqr import TvLqr, get_solver
+
+class IrsLqr():
+    def __init__(self, system, Q, Qd, R, x0, xd_trj, u_trj_initial, 
```

Comment:
I suggest that we bundle `Q`, `Qd`, `R`, `x0`, `xd_trj` and `u_trj_initial` into a structure called `IrsLqrParameters`, like this:
```python
class IrsLqrParameters:
    def __init__(self):
        self.Q = None
        self.R = None
        # etc.

```
It not only makes the list of constructor arguments shorter, it also makes it easier to add fields to `IrsLqrParameters`. 

For instance, in quasistatic contact simulations, it is better to represent the diagonal Q matrix as a dictionary of smaller matrices keyed by `ModelInstanceIndex`, as the entries in a `MultibodyPlant`'s state vector corresponding to one model instance are not always contiguous. So in the quasistatic case, I'd pass `Q_dict` as part of `IrsLqrParams` into the constructor of `IrsLqr`, and compute the Q matrix inside the constructor using a function in the `system`.

Reply:
Done. I also added an `all.py` module because the imports were getting complicated!